app/assets/javascripts/graphs.js

```diff
@@ -51,6 +52,33 @@ function zoomOut(instance) {
   });
 }
 
+function scaleUp(instance) {
+  var nodes = instance.graph.nodes();
+  var max = Math.max(...nodes.map(x => x.size));
+  nodes.forEach(x => {
+    if (isFinite(Math.log(x.size + 1))) {
```

Comment:
I found the scaling up and down a bit too gradual - it was hard to notice anything was happening.

Changing the `x.size +1` etc. values to `+ 2` makes it a more vivid change, and I think is still granular enough to be useful. But I'm also happy to be convinced about using `+1`. 😄